dashboard/src/components/AppView/AccessURLTable/AccessURLTable.tsx

```diff
@@ -1,49 +1,76 @@
 import * as React from "react";
 
-import { IResource, IServiceSpec } from "../../../shared/types";
+import LoadingWrapper from "../../../components/LoadingWrapper";
+import { IKubeItem, IResource, IServiceSpec } from "../../../shared/types";
+import isSomeResourceLoading from "../helpers";
 import AccessURLItem from "./AccessURLItem";
 import { GetURLItemFromIngress } from "./AccessURLItem/AccessURLIngressHelper";
 import { GetURLItemFromService } from "./AccessURLItem/AccessURLServiceHelper";
 
 interface IServiceTableProps {
-  services: { [s: string]: IResource };
-  ingresses: { [i: string]: IResource };
+  services: Array<IKubeItem<IResource>>;
```

Comment:
if we have IService and IIngress, could this be `Array<IKubeItem<IService | IIngress>>`? would help us in not having to do the type conversion below.